cwatm/hydrological_modules/water_quality/erosed.py

```diff
@@ -39,15 +40,107 @@ class waterquality_erosed(object):
     def __init__(self, model):
         self.var = model.var
         self.model = model
+        self.waterquality_vars = waterquality_vars(model)
 
     def initial(self):
         """
                 INITIAL PART OF THE EROSED MODULE
+
+        Sediment yield per grid cell is calculated with the Modified Universal Soil Loss Equation (MUSLE)
+        Williams (1995)
         """
-        print('erosed init')
+        self.waterquality_vars.initial()
+        # load initial MUSLE maps
+        # exponential function for fraction of rock (exp-0.053 * fcr)
+        self.var.fcfr = 1
+
+        # K_usle: USLE soil erodibility factor
+        self.var.kFactor = self.var.Kf = loadmap('kFactor')
+
+        # C_usle: USLE cover and management factor
+        self.var.cFactor = loadmap('cFactor')
+
+        # ls_usle: USLE topographic factor (slope-length)
+        self.var.lsFactor = loadmap('lsFactor')
+
+        # manning overland rougness: values for landcoverclasses from settingsfile
+        overlandManningVars = ['manForest', 'manGrassland', 'manirrPaddy', 'manirrNonPaddy']
+        self.var.manOverland = np.tile(globals.inZero, (4, 1))
```

Comment:
I recommend we create and sum 2D arrays in waterquality_main. Summation is there unless the summed array is needed earlier during executation

Reply:
Do I understand this correctly:
Line 67 and 68 go to waterquality_main.py initial?

Reply:
You can leave it here for now